h/static/styles/app.scss

```diff
@@ -82,27 +82,82 @@ ol {
 
 /* The groups dropdown list. */
 
-$group-list-width: 225px;
+$group-list-width: 270px;
 
-.group-list .dropdown {
-  white-space: nowrap;
-}
+.group-list {
+  .dropdown {
+    white-space: nowrap;
+  }
 
-.group-list .dropdown-menu {
-  width: $group-list-width;
-}
+  .dropdown-menu {
+    width: $group-list-width;
 
-.group-list .dropdown-menu li {
-  @include pie-clearfix;
-}
+    .group-name {
+      overflow: hidden;
+      text-overflow: ellipsis;
+      width: $group-list-width - 30px;
+    }
+  }
+
+  .group-item {
+    display: flex;
```

Comment:
You still need to address this. For an example, see [this commit](https://github.com/hypothesis/h/commit/3039032).